src/index.html

```diff
@@ -22,6 +22,377 @@
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price-box">
+              <p class="card__price-label">Price:</p>
+              <span class="card__price-value">$2,199</span>
+            </div>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
+          >
+            Buy
+          </button>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="Product"
+          />
+          <div class="card__content">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price-box">
+              <p class="card__price-label">Price:</p>
+              <span class="card__price-value">$2,199</span>
+            </div>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
+          >
+            Buy
+          </button>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="Product"
+          />
+          <div class="card__content">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price-box">
+              <p class="card__price-label">Price:</p>
+              <span class="card__price-value">$2,199</span>
+            </div>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
```

Comment:
The `data-qa` attribute here is `hover`, which is inconsistent with the `card-hover` used in another button. Ensure consistency in `data-qa` attributes unless different identifiers are needed for testing purposes.